docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,199 @@ which groups these values were observed, as well as the difference and ratio
+yet important difference between both contexts is the difference in skills required to
+succeed in both industries. A slightly more subtle difference is the demographic differences
+in number of genders who typically work in each of these industries, which may result from
+wording in job postings, social constructs on gender and societal roles, and the male-female
+ratio of successful applicants in each field.
+
+The Framing Trap
+^^^^^^^^^^^^^^^^
+
+This trap occurs when we fail to consider the full picture surrounding
+a particular social context when abstracting a social problem. Elements
+involved include but are not limited to: the social landscape that the
+chosen phenomenon exists in, characteristics of individuals or circumstances
+of the chosen situation, third parties involved along with their circumstances,
+and the task that is being set out to abstract (i.e. calculating a risk score,
+choosing between a pool of candidates, selecting an appropriate treatment, etc).
+
+To help us avoid drawing narrow boundaries of what is considered in scope for
+the problem, we might consider different "frames" to use.
+
+1. The algorithmic frame: these are choices made when abstracting a problem
+in the form of representations (data) and labelling (outcome). In this frame
+an algorithm is evaluated based on properties of output in relation to input.
+Examples of these properties include accuracy and generalizability to data the model
+did not train on. Fairness is typically not evaluated in this frame.
+
+2. The data frame: this frame request to the content of input and output of the
+algorithm ('data') whether in the form of abstractions, or outcomes. This additional
+frame allows us to question the inherent (un)fairness present in input and output data.
+
+3. The sociotechnical frame: this frame recognizes that a machine learning model is part
+of the interaction between people and technology, and thus any social components of this
+interaction need to be modelled and incorporated accordingly.
+
+Example: assessing risk of re-engagement in criminal behaviour in an individual
+charged with an offense, and appropriate measures to prevent relapse, and failing
+to consider factors such as race, socio-economic status, mental health, along with
+socially-dependent views present in judges, police officers, or any actors responsible
+for recommending a course of action.
+
+In the algorithmic framework, for example, input variables may contain previous criminal history,
+statements taken by the accused, witnesses and police officers. Labels (outcome)
+include recommendations by the algorithm on an appropriate course of action based
+on a computed risk score. Model is limited in assessing fairness out outcome.
+
+The data framework could attempt to reduce unfairness by studying socio-economic
+information regarding the accused, their upbringing and how it relates to their
```

Comment:
For simplicity to get this approved, we could remove the two paragraphs "In the algorithmic framework" and "The data framework could..." and only keep the sociotechnical framework paragraph below (since that's what we're hoping people will take anyway)

Reply:
I agree with @mmadaio . I don't think we should encourage people to look at anything but the sociotechnical frame because it suggests that it's sufficient to do so.

Reply:
Sounds good!

Reply:
Removed paragraphs